applications/GeoMechanicsApplication/tests/lysmer_boundary_column2d_quad/MeshStructure.svg

```diff
@@ -0,0 +1,3382 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
Very nice pictures!

The roller conditions are placed exactly on the top nodes. By convention that would mean that they only apply to that node. To let them apply for an entire edge, it is common to place them on an edge ( not exactly on a node ) and repeat that once or twice further along the edge

This remark is similar for the other picture.

Reply:
Moved rollers and added more of them.